shimmer/develop/what-is-shimmer/tokens-and-wallets.md

```diff
@@ -0,0 +1,52 @@
+---
+description: 'Learn about the role of the shimmer token, storage deposits, where to buy SMR and the recommended wallets.'
```

Comment:
Should this be part of the dev section?

Reply:
I'm iffy on this as well, But if you want to work outside the testnet, you are going to need tokens, and it's a nice place to mention that you can get as many as you want from the faucet.

Reply:
But will a dev use firefly or tanglepay for example? For me this just feels like double the work.
Can't we just add something like usefull links at the bottom with buying and the wallets?

Reply:
No you are right, changing it to just links doesn't make much sense.
Still I'm not sure if a dev would ever be interested in wallets, but especially in buying Tokens 🤔

Reply:
But we can always try keeping this page. If we don't like it, we can just remove/change it later